mobile/native/src/ln_dlc/mod.rs

```diff
@@ -930,7 +884,8 @@ fn update_state_after_collab_revert(
                 direction: position.direction.opposite(),
                 order_type: OrderType::Market,
                 state: OrderState::Filled {
-                    execution_price: execution_price.to_f32().expect("to fit into f32"),
+                    // the price doesn't matter here. It's just to clean up some state
+                    execution_price: 21.0,
```

Comment:
Even if its just a clean up, it would have been nice to keep the execution price for reference. This has typically been discussed with the user, if we had to run a collab revert. I don't see the improvement in now showing `21` as execution price for these collab reverts.

But yet again, why do we even create an order for that? 😅 I think it shouldn't be needed anymore now that we've moved away from our ominous reliance on a specific order of our orders to make our UI work 🙈

Reply:
Oh, I didn't know we show this information 🙈

Well, for that to work, we need to define the execution price in the collab_revert endpoint and send it to the user.